anomalib/deploy/inferencers/torch.py

```diff
@@ -138,8 +138,12 @@ def post_process(
             anomaly_map = predictions
             pred_score = anomaly_map.reshape(-1).max()
         else:
-            anomaly_map, pred_score = predictions
-            pred_score = pred_score.detach().cpu().numpy()
+            if isinstance(predictions[1], (Tensor)):
+                anomaly_map, pred_score = predictions
+                pred_score = pred_score.detach()
```

Comment:
I don't get this. If scores are of type `Tensor` then we should detach and convert to numpy right? And in the else statement, we shouldn't do anything.

Reply:
I haven't added anything myself. this is taken from #67 . Just created this PR so that you could address it, if you think it's not correct:)

Reply:
Okay, thanks for creating this. I'll look into it.

Reply:
The fix is for patchcore, which returns both anomaly map and score after `forward`. Without this check, pred_score is converted to numpy, but not anomaly_map. That's why we see type mismatch error in normalize function.

I think this would be an enough fix for now. We'll use PL predict stuff anyway for torch inference.